### DISCUSSION TITLE
Booking approved email not sent though reservation mail is sent

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#573] [General]
Hi, somehow Booking approved emails are not sent from https://ihbv.nl/evenementen/hb-ondernemers-hackaton-17-september-2024/. 
Reservation emails are sent, but when customers pay, they don't receive the Booking approved email.
I checked settings and compared them to an other site that does send the booking approved emails. 
Event is set on automatic approval after payment. Payment emails not activated, approved booking emails activated. 
I did what I always do but somehow the Booking approved emails are not sent. 
Can you help?

**Comment by liedekef** [comment]
Have you checked if EME attempts to send the mails? You can do this in the submeny "E-mail beheer", tab "Verzonden e-mails"
All mails get added there, so that's the first place to look ...
Also: do the bookings get approved after online payment (in other words: is the payment gateway handling working ok?).

**Comment by JanineKall** [comment]
Thanks, I checked, last time a Booking approved mail was sent was May 22nd. 
All payments ever since have been accepted by Mollie, bookings have also been approved in EME. But no confirmation emails, only reservation emails. 
So May 22nd afternoon it suddenly stopped sending Booking Approved emails.

 

Janine Kallenbach

 

Van: Franky Van Liedekerke ***@***.***> 
Verzonden: vrijdag 7 juni 2024 14:49
Aan: liedekef/events-made-easy ***@***.***>
CC: JanineKall ***@***.***>; Author ***@***.***>
Onderwerp: Re: [liedekef/events-made-easy] Booking approved email not sent though reservation mail is sent (Discussion #573)

 

Have you checked if EME attempts to send the mails? You can do this in the submeny "E-mail beheer", tab "Verzonden e-mails"
All mails get added there, so that's the first place to look ...
Also: do the bookings get approved after online payment (in other words: is the payment gateway handling working ok?).

**Comment by liedekef** [comment]
Do you have the option set to send these mails? See E-mail options:
![image](https://github.com/liedekef/events-made-easy/assets/75667/82750e12-8d01-4956-8bc1-2fa1b5d1e13f)

**Comment by JanineKall** [comment]
Yes, I did:




	
		


Thing is, problem occurred during the event. 
We had several bookings that went well, but after the last booking with approved RSVP email on May 22nd at 12.41, 
only reservation bookings were sent, while I changed nothing. Payments were all succesful so clients were asking questions😉



 

I notices there is a new update available. Would updating help?

 

ir. Janine Kallenbach - Heinen

 

Bestuurder Instituut Hoogbegaafdheid Volwassenen | Voorzitter RvT stichting iQ+ 
TMA-specialist | Websitebouwer | Trainer persoonlijke vaardigheden | Partner TalentCaching
Lid Wijkraad Lage Land Prinsenland 's Gravenland Rotterdam 

 

 <https://www.linkedin.com/in/janine-kallenbach-heinen/> LinkedIn

 ***@***.***> ***@***.***  

 <tel:+31622281871> +31 6 222 81 871 

 

 

Van: Franky Van Liedekerke ***@***.***> 
Verzonden: vrijdag 7 juni 2024 17:30
Aan: liedekef/events-made-easy ***@***.***>
CC: JanineKall ***@***.***>; Author ***@***.***>
Onderwerp: Re: [liedekef/events-made-easy] Booking approved email not sent though reservation mail is sent (Discussion #573)

 

Do you the option set to send these mails? See E-mail options:
image.png (view on web) <https://github.com/liedekef/events-made-easy/assets/75667/82750e12-8d01-4956-8bc1-2fa1b5d1e13f>

**Reply by liedekef** [reply]
it means there is a combo of settings causing the mail not to be sent, if not present in the EME queue.
I don't have all the info, so either you make me admin so I can check or you check option per option (and make sure no custom code is interfering)

**Reply by liedekef** [reply]
no further reply, so considering solved.
